core/data/pressure_graph/manager.py

```
@@ -987,4 +1033,4 @@ def increase_pressure(self, node_id: str, amount: float, reason: str) -> None:
             self.recompute_pressure()
             
         except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)